paasta_tools/paasta_metastatus.py

```diff
@@ -39,6 +40,11 @@
 from paasta_tools.utils import PaastaColors
 from paasta_tools.utils import print_with_indent
 
+
+logging.basicConfig()
+# kazoo can be really noisy - turn it down
+logging.getLogger("kazoo").setLevel(logging.CRITICAL)
```

Comment:
Was this info before? I agree it's annoyingly chatty, maybe ERROR would be okay though.